website/src/main/tut/docs/rules/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: docs
+title: Rules
+---
+
+# Rules
+
+Scalafix comes with a few rules out-of-the-box.
+These rules have been chosen to meet the long-term goal of scalafix to
+[clarify the Scala to Dotty migration path](http://scala-lang.org/blog/2016/05/30/scala-center-advisory-board.html#the-first-meeting).
+
+Here's a list of the currently implemented rules:
+
+{% assign rules = site.data.menu.options | where:"title","Rules" %}
+{% for rule in rules[0].nested_options %}
+  - [{{ rule.title }}]({{ rule.title }}.html)
```

Comment:
If we remove the subsection for each rule in the main menu, can we still automate this for example by listing `docs/rules/*.md`?

Reply:
we can do something like that, I'm pretty sure.